pages/k8s/install-manual.md

```diff
@@ -215,13 +212,39 @@ juju deploy charmed-kubernetes
 It is also possible to deploy a specific version of the bundle by including the `channel` argument to select the desired version and risk level. For example, to deploy the **Charmed Kubernetes** bundle for the Kubernetes 1.25 release, you could run:
 
 ```bash
-juju deploy --channel=1.25/stable
+juju deploy charmed-kubernetes --channel=1.25/stable
 ```
 
 This also applies to channels which have not yet had a stable release, for example:
 
 ```bash
-juju deploy --channel=1.27/beta
+juju deploy charmed-kubernetes --channel=1.27/beta
+```
+
+## Deploying an older Kubernetes version with newer charms
+
+Each Charmed Kubernetes release supports N-2 kubernetes releases.
+For example, Charmed Kubernetes 1.26 supports 1.26, 1.25, and 1.24 binary
+versions. Therefore, if you wish to install an older version of Kubernetes
+you may still be able to do so with the latest stable charms. 
+
+Let's say, your project requires Kubernetes 1.24. Build an overlay which adjusts
+the charm configuration for that revision
+
+```yaml
+applications:
+  kubernetes-control-plane:
+    options:
+      channel: 1.24/stable
+  kubernetes-worker:
+    options:
+      channel: 1.24/stable
+```
+
+and deploy with the 1.26/stable charm bundle
+
+```bash
+juju deploy charmed-kubernetes --channel=1.26/stable --overlay=1.24_overlay.yaml
 ```
```

Comment:
Add instructions as to how to deploy an older k8s version with the latest charms

Reply:
:cake: